core-bundle/src/Resources/contao/languages/en/tl_page.xlf

```diff
@@ -300,7 +300,7 @@
         <source>Two-factor redirect page</source>
       </trans-unit>
       <trans-unit id="tl_page.twoFactorJumpTo.1">
-        <source>Please choose the page to which visitors will be redirected when setting up two-factor authentication.</source>
+        <source>Please choose the page to which members will be redirected to set up the two-factor authentication.</source>
```

Comment:
```suggestion
        <source>Please choose the page to which members will be redirected to set up two-factor authentication.</source>
```

To me, "the two-factor authentication" sounds a little strange. Not 100% sure what is correct here though. Maybe @madmaharaja can help?

Reply:
"... set up two-factor authentication." is correct in this case (without "the").